spec/slack/real_time/client_spec.rb

```diff
@@ -159,16 +159,31 @@
       end
 
       describe '#run_handlers' do
-        context 'when store has no event hooks' do
+        before do
+          @events = client.store.class.events.dup
+          client.store.class.events.clear
+        end
+
+        after do
+          client.store.class.events.merge!(@events)
```

Comment:
I know this was copied, but should this be an assignment? Otherwise you inherit events added during the test.

Reply:
that was just lifted from the prior code but i can change it. i have no idea what the expected performance is, i just re-used what was there.

Reply:
making this an assignment instead of a merge broke... a bunch of other tests. so i guess this is necessary

Reply:
Yep, this was written because `events=` doesn't exist. It could be `client.store.class.events.instance_variable_set(:@events, ...)`, but a cleaner fix would be to implement a `reset` method on the store just like for config. Either way that's outside of the scope of this PR.